client/flutter/lib/pages/home_page.dart

```diff
@@ -26,10 +28,19 @@ class _HomePageState extends State<HomePage> {
 
   void _initStateAsync() async {
     var onboardingComplete = await UserPreferences().getOnboardingCompleted();
+
+    // TODO: Uncomment for testing.  Remove when appropriate.
```

Comment:
For whomever is working on these - Uncomment this line to ignore the "completed" state and always show the screens on launch.